index.bs

```diff
@@ -1136,6 +1138,13 @@ Methods</h4>
 		is via its promise return value, the callback parameters are
 		provided for legacy reasons.
 
+		<div class="correction" id="c2361-1">
+		  <span class="marker">Candidate Correction
+		  <a href="https://github.com/WebAudio/web-audio-api/issues/2361">Issue 2361-1.</a>
+		  </span>
+		  Use new Web IDL buffer primitives
```

Comment:
Since there are 7 related changes, it might be useful to add buttons to move to the next (and previous) corrections.  This clutters things up a bit, but makes it easier to visit all the related changes.  Or we can just expect the user to look at the changelog to find all the related corrections, where we can generate the list automatically.

Reply:
Without the buttons, finding related changes is harder.  We do have the ChangeLog which has a list of the changes, but having to go back and forth to the change log makes looking at the changes harder.

See also https://github.com/WebAudio/web-audio-api/pull/2366#issuecomment-878378045


Not sure what you mean by "new practice" and what part of the change you're referring to.